.gitignore

```diff
@@ -0,0 +1,61 @@
+# OS X generated file
```

Comment:
This is taken from [WooCommerce Android](https://github.com/woocommerce/woocommerce-android/blob/develop/.gitignore) with some entries removed (as not relevant).